lib/ae_mdw/db/mutations/stats_mutation.ex

```diff
@@ -76,6 +79,14 @@ defmodule AeMdw.Db.StatsMutation do
       nil ->
         Model.stat(index: Stats.max_tps_key(), payload: {tps, key_hash})
     end)
+    |> then(fn state ->
+      time =
+        key_hash
+        |> :aec_db.get_block()
+        |> :aec_blocks.time_in_msecs()
+
+      SyncStats.increase_statistics(state, :contracts, time, contracts_created)
```

Comment:
This looks like it belongs on the `BlockStatisticsMutation` instead of here, can we move it there?

Reply:
If we move it to the BlockStatisticsMutation we can re-use `increment_statistic/3`, we also wouldn't need to re-fetch the block too get the time

Reply:
Yes, but then we would need to calculato how many contracts have been created, since we wouldn't know if the stats in the state have been updated yet

Reply:
That'd be better, passing the time